DependencyInjection/Configuration.php

```diff
@@ -41,6 +41,7 @@ public function getConfigTreeBuilder()
                     ->prototype('scalar')->end()
                 ->end()
                 ->scalarNode('active_theme')->defaultNull()->end()
+                ->scalarNode('redirect_fallback')->defaultValue('homepage')->end()
```

Comment:
`homepage` may not exist for other people who are using this bundle, it's better to put `/`

Reply:
Okay, fair point, I will change that to use null as default as prepared in ThemeController.php already.

Reply:
Done.